packages/flutter/lib/src/widgets/media_query.dart

```diff
@@ -756,7 +756,9 @@ class MediaQueryData {
 /// Establishes a subtree in which media queries resolve to the given data.
 ///
 /// For example, to learn the size of the current media (e.g., the window
-/// containing your app), you can read the [MediaQueryData.size] property from
+/// containing your app), you can use [MediaQuery.sizeOf]: 
+/// `MediaQuery.sizeOf(context)`. Alternatively, with less performance,
```

Comment:
Let's remove the second sentence (starting with "Alternatively") altogether. We don't have to show people how to do something in a less then optimal way.

Reply:
Instead, we should also update the next two paragraphs to also cover `sizeOf` and friends and maybe document _why_ it is better to use a specific one.